android/java/org/chromium/chrome/browser/preferences/BravePrefServiceBridge.java

```diff
@@ -13,6 +13,14 @@
 import org.chromium.base.annotations.NativeMethods;
 import org.chromium.chrome.browser.profiles.Profile;
 
+/**
+ * Please don't add anything in that file. We are going to remove it soon.
+ * Check this PRs on how to handle preferences correctly:
+ * https://github.com/brave/brave-core/pull/16356
+ * https://github.com/brave/brave-core/pull/15905
+ * Contact code owners if you have additional questions.
```

Comment:
I'm going to add here that local_state pref can still go to that bridge for now.